src/Type/Definition/InputObjectType.php

```diff
@@ -45,16 +48,12 @@ public function __construct(array $config)
     }
 
     /**
-     * @param string $name
-     *
-     * @return InputObjectField
-     *
-     * @throws Exception
+     * @throws InvariantViolation
      */
-    public function getField($name)
+    public function getField(string $name) : InputObjectField
     {
-        if ($this->fields === null) {
-            $this->getFields();
+        if (! isset($this->fields)) {
```

Comment:
Those checks are pretty repetitive, we might move them into `initializeFields()`. Not sure if the cost of always calling a method is significant and in a hot path.

Reply:
@vladar what is your opinion here?

Reply:
I would keep it as is.